docs/release-notes/known-issues.md

```diff
@@ -4,6 +4,18 @@ navigation_title: "Known issues"
 
 # Kibana known issues
 
+::::{dropdown} Errors in rule executions occur when maintenance windows have filters
+
+Applies to: {{stack}} 9.0.0, 9.0.1
+
+**Details** 
+Errors occur when rules execute during an active maintenance window that has filters and a matching rule category. 
```

Comment:
We've generally being trying to replace these terms. Likewise title could be "Errors when rules run during maintenance windows with filters"
```suggestion
Errors occur when rules run during an active maintenance window that has filters and a matching rule category. 
```